File: code/charlie/Python/Blackjack.py
########### Lab 14 - Blackjack Advice ##############

def main():

    choice = 'yes'
    total = 0
    cards = ''
    print( 'let\'s play some Backjack shall we?!')

    while choice == 'yes':
        
        for i in range(3):
            if i == 0:
                cards = input('what\'s your first card? ')
            elif i == 1:
                cards = input('what\'s your second card? ')
            else:
                cards = input('what\'s your third card? ')

            if cards == 'A':
                total += 1
                continue
            if cards == 'J' or cards == 'Q' or cards == 'K':
                total += 10
                continue
        
            cards = int(cards)
            total += cards
            cards = str(cards)
            
        if total < 17:
            print('Hit!')
        elif total >= 17 and total < 21:
            print('Stay')
        elif total == 21:
            print('Blackjack!!!')
        else:
            print('Already busted!')
        choice = input('Would you like to play again (yes/no)? ')
        total = 0

main()

File: code/charlie/JavaScript/lab1_p1/blackjack.js
let choice = 'yes';
let total = 0;
let card = '';
alert('Let\'s play some blackjack shall we?!')

while(choice === 'yes'){

    for(let i = 0; i < 3; ++i){
        if(i === 0){
            card = prompt('What\'s your first card?');
        } else if(i === 1){
            card = prompt('What\'s your second card?');
        } else{
            card = prompt('What\'s you third card?');
        }

        if(card === 'A'){
            total += 1;
        } else if(card === 'J' || card === 'Q' || card === 'K'){
            total += 10;
        } else{
            total += parseInt(card)
        }
        console.log(total);
    }
    if(total < 17){
        alert('Hit!');
    } else if(total >= 17 && total < 21){
        alert('Stay!');
    }else if(total === 21){
        alert('Blackjack!!!!');
    } else{
        alert('Ya already busted sucka!');
    }
    choice = prompt('Would you like play again (yes/no)?');
    total = 0
}